backend/event-parse/parse.go

```diff
@@ -78,6 +78,7 @@ const (
 
 const (
 	ScriptPlaceholder = "SCRIPT_PLACEHOLDER"
+	PROXY_URL         = "https://replay-cors-proxy.highlightrun.workers.dev"
```

Comment:
nit: could you rename this to `ProxyUrl`?
https://stackoverflow.com/questions/22688906/go-naming-conventions-for-const

Reply:
Done.